### PR TITLE
Unified test documentation: security module

### PR DESCRIPTION
**Pull request by sulewskiprzemyslaw**
Signed-off-by: Przemysław Sulewski <przemyslaw.sulewski@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-security/203-measured-boot.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+TPM: Digest of FMAP: FW_MAIN_A CBFS: vbt.bin to PCR 2 measured
+TPM: Digest of FMAP: FW_MAIN_A CBFS: fallback/dsdt.aml to PCR 2 measured
+TPM: Digest of FMAP: FW_MAIN_A CBFS: fallback/payload to PCR 2 measured
+```
+
+The output should also not contain the following message:
+
+```
+TPM: Extending hash into PCR failed.
+```
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-security/203-measured-boot.md:66]:
```suggestion
```

```

Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-security/203-measured-boot.md:66]:
```suggestion
```


```

Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-security/203-measured-boot.md:66]: no newline at the end of the file (I cannot add a proper suggestion)

File: docs/unified-test-documentation/dasharo-security/206-secure-boot.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+1. Select the `Reset` option to apply the settings and reboot.
+1. The DUT will now attempt to boot `OPERATING_SYSTEM` with Secure Boot enabled.
+1. Log in to the default user session.
+1. Press Windows+R to open Run Window. Type msinfo32 and press Enter.
+1. In the System Information window, go to System Summary, and in the right pane
+   select the Secure Boot State and check its state.
+
+**Expected result**
+
+The Secure Boot State should be enabled.
```
Comment (damiankaras) [inline docs/unified-test-documentation/dasharo-security/206-secure-boot.md:95]: no newline at the end of the file (I cannot add a proper suggestion)

**Review by damiankaras — COMMENTED** [review]
I left suggestions

File: docs/unified-test-documentation/dasharo-security/203-measured-boot.md (hunk 1 above)
Comment (macpijan) [reply docs/unified-test-documentation/dasharo-security/203-measured-boot.md:66]: This pre-commit rule already checks for the proper fine ending: https://github.com/Dasharo/docs/blob/master/.pre-commit-config.yaml#L11

Comment (macpijan) [reply docs/unified-test-documentation/dasharo-security/203-measured-boot.md:66]: This is invalid, IMHO.

**Pull request merged**